native/cocos/editor-support/spine-wasm/spine-skeleton-instance.cpp

```diff
@@ -355,20 +353,20 @@ void SpineSkeletonInstance::collectMeshData() {
         SpineMeshData::moveIB(currMesh.iCount);
         // record debug shape info
         if (_userData.debugMode) {
-            SpineDebugShape debugShape;
+            size_t currentShapesLen = _debugShapes.size();
+            _debugShapes.setSize(currentShapesLen + 1, {});
+            SpineDebugShape& debugShape = _debugShapes[currentShapesLen];
```

Comment:
Avoid creating temporary `SpineDebugShape` object and copy to spine::Vector<>.
Create it in spine::Vector<> and get the reference to modify its content will make the binary smaller since less wasm directives be generated.